web_monitoring/html_diff_render.py

```diff
@@ -435,6 +371,304 @@ def _count_changes(opcodes):
     }
 
 
+# --------------------- lxml.html.diff Tokenization --------------------------
```

Comment:
Not reviewing any of this section. Let me know if you want me to look at it.

Reply:
The parts I modified (for undiffable stuff) are [564-569](https://github.com/edgi-govdata-archiving/web-monitoring-processing/pull/152/files#diff-f483e5fdf85585c0850051651929f289R564) and [608-611](https://github.com/edgi-govdata-archiving/web-monitoring-processing/pull/152/files#diff-f483e5fdf85585c0850051651929f289R564), if you want to look at that.